src/profile-logic/tracks.js

```diff
@@ -1100,10 +1103,28 @@ export function computeDefaultVisibleThreads(
   // We keep all threads whose sampleScore is at least 5% of the highest
   // sampleScore, and also any threads which are otherwise essential.
```

Comment:
nit: please add to this comment a mention of removing the parent process threads when requested.